constraints.txt

```diff
@@ -248,6 +251,7 @@ tzlocal==2.1
 urllib3==1.25.10
 validators==0.18.2
 virtualenv==20.0.33
+watchdog==2.1.2
```

Comment:
`watchdog` uses and Apache license: https://github.com/gorakhargosh/watchdog/blob/master/LICENSE.